packages/manager/apps/container/src/container/nav-reshuffle/onboarding/walkMe/index.tsx

```diff
@@ -146,6 +146,13 @@ export const OnboardingWalkMe = () => {
     }
   };
 
+  const resizeObserver = new ResizeObserver((entries) => {
+    entries.forEach((entry) => {
+      const el: HTMLElement = stepElement.current;
+      el.style.height = `${entry.borderBoxSize[0].blockSize + ELEMENT_OFFSET}px`;
+    })
```

Comment:
Maybe setup a debounce so that we do not loop over the `entries` too much

Reply:
I update the function with a find in place of the forEach, a little more efficient. Also I check and mainly the function Is called just once and there is in the entries only the observed elements. So it should be ok in term of performance.